frontend/src/features/permits/pages/TermOversize/components/review/ConfirmationCheckboxes.tsx

```diff
@@ -16,16 +16,16 @@ export const ConfirmationCheckboxes = ({
   const checkboxes = [
     {
       description:
-        "I confirm that this permit is issued to the registered owner (or lessee) of the vehicle being permitted.",
+        "Confirm that this permit is issued to the registered owner (or lessee) of the vehicle being permitted.",
       checked: false,
     },
     {
       description:
-        "I confirm I am compliant with the appropriate policy for my selected commodity(s).",
+        "Confirm compliance with the appropriate policy for the selected commodity(s).",
       checked: false,
     },
     {
-      description: "I confirm the information in this application is correct.",
+      description: "Confirm the information in this application is correct.",
```

Comment:
is it worth putting these 3 strings into the `constants.ts` file?

Reply:
For the moment I think it's fine where it is. In the future if it's gonna be used elsewhere then I think we should move it to constants.ts.